actionpack/lib/action_dispatch/http/url.rb

```diff
@@ -101,10 +101,10 @@ def extract_subdomains_from(host, tld_length)
         end
 
         def add_trailing_slash(path)
-          # includes querysting
+          # Includes querysting.
```

Comment:
This doesn't really count as a sentence and doesn't really add anything to comprehension of the code. Either delete the comment or expand on it to explain why it's doing what it's doing.